packages/restapi/src/lib/chat/send.ts

```diff
@@ -26,18 +26,25 @@ export const send = async (options: Omit<ChatOptionsType, 'connectedUser'>) => {
     if (!isValidETHAddress(account)) {
       throw new Error(`Invalid address!`);
     }
-    if (!isValidETHAddress(receiverAddress)) {
-      throw new Error(`Invalid address!`);
+
+    let isGroup = true;
+    if (receiverAddress.includes('eip155:')) {
```

Comment:
Can you please explain more on this condition?

Reply:
if (isValidETHAddress(receiverAddress)) {
        isGroup = false;
      }
else
{
  throw new Error(`Invalid address!`);
}